toolkit/imageconfigs/scripts/cleanup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
```

Comment:
Please add the license banner
`
# Copyright (c) Microsoft Corporation.
# Licensed under the MIT License.
`

Reply:
good reminder. 

I see other scripts under `postinstallscripts` don't have it though. so wondering what's the criteria for that?

Reply:
AFAIK, all scripts should be carrying it